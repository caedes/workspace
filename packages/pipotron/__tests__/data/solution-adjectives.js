import solutionAdjectives from '../../src/data/solution-adjectives';

describe('solutionAdjectives', () => {
  it('returns the solutionAdjectives', () => {
    expect(solutionAdjectives).toMatchSnapshot();
  });
});
