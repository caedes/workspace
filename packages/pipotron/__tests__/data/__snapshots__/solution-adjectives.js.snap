// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`solutionAdjectives returns the solutionAdjectives 1`] = `
Array [
  "imaginables",
  "possibles",
  "s'offrant à nous",
  "de bon sens",
  "envisageables",
  "éventuelles",
  "réalisables",
  "déclinables",
  "pertinentes",
  "que nous connaissons",
  "évidentes",
  "optimales",
  "opportunes",
  "emblématiques",
  "draconiennes",
]
`;
